### ISSUE TITLE
Native Ad Support (not Native Express)

### ISSUE THREAD

**Issue by Skintillion** [EddyVerbruggen/nativescript-admob#25]
Is there any support for Native ads?

https://developers.google.com/admob/android/native

Thanks!

**Comment by EddyVerbruggen** [comment]
Not at the moment. If anyone else needs this, please give a 👍 reaction on the OP's post.

**Comment by calebeaires** [comment]
@EddyVerbruggen 
**Native Ads increase earning in 10%. This feature will be a great addition!** I have done a [job with native](https://github.com/EddyVerbruggen/nativescript-admob/issues/11), but it is not a good one. Help us 👍

**Comment by brownkevg** [comment]
@EddyVerbruggen 
I have a very rough implementation of this working for android in the demo project here [https://github.com/brownkevg/nativescript-plugin-firebase/tree/ui](url). Its currently forked from your nativescript-plugin-firebase repo but I assume that it could be applied to this repo as well. It still needs some work but I'm looking to make a PR for this. If thats something you would like let me know.

**Comment by EddyVerbruggen** [comment]
Yes @brownkevg that would be awesome! I can monkeypatch that PR over to this repo as well of course.

**Comment by brightamegbor** [comment]
> @EddyVerbruggen
> I have a very rough implementation of this working for android in the demo project [here](https://github.com/brownkevg/nativescript-plugin-firebase/tree/ui). Its currently forked from your nativescript-plugin-firebase repo but I assume that it could be applied to this repo as well. It still needs some work but I'm looking to make a PR for this. If thats something you would like let me know.

Hello, I followed your docs on this from your repo, but couldn't figure out how to implement using nativescript angular. I got nativeads return: 5 in console but couldn't display the ads in my template

**Comment by tbenoist1** [comment]
Is there any planned update for native ads ?

**Comment by NazimMertBilgi** [comment]
> Is there any planned update for native ads ?

+1

**Comment by dungfv** [comment]
> Is there any planned update for native ads ?

+1
